attributes/default.rb

```diff
@@ -5,7 +5,12 @@
 default[:omnibus_updater][:remove_chef_system_gem] = false
 default[:omnibus_updater][:prerelease] = false
 default[:omnibus_updater][:disabled] = false
-default[:omnibus_updater][:kill_chef_on_upgrade] = true
+default[:omnibus_updater][:upgrade_behavior] = 'exec'
+default[:omnibus_updater][:upgrade_notification] = :immediately
```

Comment:
This `upgrade_notification` attribute seems to be unused, was this made obsolete by the latest commit in this PR? The PR comment states that "The original behavior is still accessible via attributes." but from reading the code here it looks like this is no longer the case